src/transactions/utils/validation.js

```diff
@@ -63,13 +63,11 @@ export const validateAddress = address => {
 	}
 
 	const addressAsBignum = bignum(address.slice(0, -1));
-	const biggestAddress = bignum
-		.fromBuffer(Buffer.from([255, 255, 255, 255, 255, 255, 255, 255]))
-		.plus(1);
+	const biggestAddress = bignum.fromBuffer(Buffer.from(new Array(8).fill(255)));
```

Comment:
How about defining this in `constants`?

Reply:
Yeah, I also had that thought. Lets do that then.